lib/sync.js

```diff
@@ -54,6 +57,30 @@ function Sync(src, target, opts, callback) {
       doc: doc
     });
   }
+  function pushPaused() {
+    self.pushPaused = true;
+    if(self.pullPaused) {
+      self.emit('paused');
+    }
+  }
+  function pullPaused() {
+    self.pullPaused = true;
+    if(self.pushPaused) {
+      self.emit('paused');
+    }
+  }
+  function pushActive() {
+    self.pushPaused = false;
+    self.emit('active', {
+      direction: 'push'
+    });
```

Comment:
So this will emit two `active` events but one `paused` event every time network conditions change? Why not emit a single `active` event the same way you are doing for `paused`?